src/shared/components/ncTable/partials/TableCellNumber.vue

```diff
@@ -3,44 +3,151 @@
   - SPDX-License-Identifier: AGPL-3.0-or-later
 -->
 <template>
-	<div>
-		{{ column.numberPrefix }}{{ getValue }}{{ column.numberSuffix }}
+	<div class="cell-number">
+		<div v-if="!isEditing" @dblclick="startEditing">
```

Comment:
@marcoambrosini @jancborchardt I remember double click was something we had some debates about in the past. Is this fine as a trigger for the inline editing even though it is not very common in web apps? WDYT?

Reply:
No, forgot to comment on that, should be on single click